src/OpenLoco/Localisation/StringIds.h

```diff
@@ -1752,4 +1752,5 @@ namespace OpenLoco::StringIds
     constexpr string_id shortcut_construction_build_at_current_pos = 2224;
     constexpr string_id shortcut_construction_remove_at_current_pos = 2225;
     constexpr string_id shortcut_construction_select_position = 2226;
+    constexpr string_id trains_reverse_at_signals = 2227;
```

Comment:
The string itself is missing. Could you commit your change to `en-GB.yml`?

Reply:
Oops.. Can't believe I missed that. Also added to `en-US.yml`